### PR TITLE
add DB_HOST to .env.sample

### PR DESCRIPTION
**Pull request by BolajiOlajide**
#### What Does This PR Do?
Add DB_HOST to .env.sample

#### Description Of Task To Be Completed
Add DB_HOST to .env.sample

#### Any Background Context You Want To Provide?
N/A

#### How can this be manually tested?
N/A

#### Screenshot(s)
N/A

#### Any new frontend dependency?
N/A

#### Any new backend dependency?
N/A

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -1,20 +1,20 @@
...
+  db:
     restart: always
     image: postgres
     container_name: react_lib_db
   web:
     container_name: react_lib_web
     build: ./reactlibapp
     env_file:
-      - .env
+      - ./reactlibapp/.env
```
Comment (andela-sjames) [inline docker-compose.yml]: Wait why is your `.env` not in the root, it should be in the root for consistency according to the README right?

Comment (andela-sjames) [reply docker-compose.yml]: Also the readme sample .env should be updated

Comment (andela-sjames) [reply docker-compose.yml]: I have always maintained one in the root plus my setup broke when I noticed a path mismatch some time ago and for consistency let's maintain just one in the root file

Comment (BolajiOlajide) [reply docker-compose.yml]: Okay then.

Comment (BolajiOlajide) [reply docker-compose.yml]: Done.

**Pull request merged**